src/coreclr/vm/jitinterface.cpp

```diff
@@ -140,7 +140,14 @@ FCIMPLEND
 FCIMPL1(INT64, GetCompiledMethodCount, CLR_BOOL currentThread)
 {
     FCALL_CONTRACT;
-
+    // WIP-DEBUG-ONLY change: Windows is causing problems in the CI, but can't
```

Comment:
This WIP code needs to be reverted!

Reply:
Oh I forgot! Good catch, Jan. I will submit a PR fixing that right away.